pkg/service/query.go

```diff
@@ -9,6 +9,17 @@ import (
 	"github.com/jaegertracing/jaeger-operator/pkg/util"
 )
 
+// NewQueryAdminService returns a new Kubernetes service for Jaeger Query with admin port enabled
+func NewQueryAdminService(jaeger *v1.Jaeger, selector map[string]string) *corev1.Service {
+	service := NewQueryService(jaeger, selector)
+	service.Spec.Ports = append(service.Spec.Ports,
```

Comment:
Sorry for not having caught this before, but I think we might need an extra service. I'm not sure it's a good idea to add one extra port to the services here, as people might be exposing these services currently and expect only one (or a specific set) of ports to be open.

Reply:
The same applies to the other components.

Reply:
collector, ingester and query component now create an additional Service with the admin port enabled